services/crypto_create.proto

```diff
@@ -30,147 +37,184 @@ import "basic_types.proto";
 import "duration.proto";
 
 /*
- * Create a new account. After the account is created, the AccountID for it is in the receipt. It
- * can also be retrieved with a GetByKey query. Threshold values can be defined, and records are
- * generated and stored for 25 hours for any transfer that exceeds the thresholds. This account is
- * charged for each record generated, so the thresholds are useful for limiting record generation to
- * happen only for large transactions.
- *
- * The Key field is the key used to sign transactions for this account. If the account has
- * receiverSigRequired set to true, then all cryptocurrency transfers must be signed by this
- * account's key, both for transfers in and out. If it is false, then only transfers out have to be
- * signed by it. When the account is created, the payer account is charged enough hbars so that the
- * new account will not expire for the next autoRenewPeriod seconds. When it reaches the expiration
- * time, the new account will then be automatically charged to renew for another autoRenewPeriod
- * seconds. If it does not have enough hbars to renew for that long, then the remaining hbars are
- * used to extend its expiration as long as possible. If it is has a zero balance when it expires,
- * then it is deleted. This transaction must be signed by the payer account. If receiverSigRequired
- * is false, then the transaction does not have to be signed by the keys in the keys field. If it is
- * true, then it must be signed by them, in addition to the keys of the payer account. If the 
- * auto_renew_account field is set, the key of the referenced account must sign.
+ * Create a new account.
  *
- * An entity (account, file, or smart contract instance) must be created in a particular realm. If
- * the realmID is left null, then a new realm will be created with the given admin key. If a new
- * realm has a null adminKey, then anyone can create/modify/delete entities in that realm. But if an
- * admin key is given, then any transaction to create/modify/delete an entity in that realm must be
- * signed by that key, though anyone can still call functions on smart contract instances that exist
- * in that realm. A realm ceases to exist when everything within it has expired and no longer
- * exists.
+ * If the auto_renew_account field is set, the key of the referenced account
+ * MUST sign this transaction.<br/>
+ * Current limitations REQUIRE that `shardID` and `realmID` both MUST be `0`.
+ * This is expected to change in the future.
  *
- * The current API ignores shardID, realmID, and newRealmAdminKey, and creates everything in shard 0
- * and realm 0, with a null key. Future versions of the API will support multiple realms and
- * multiple shards.
+ * ### Block Stream Effects
+ * The account identifier for the newly created account SHALL be in the
+ * transaction receipt.
```

Comment:
There is no `transaction receipt` in `block stream` right. You mean `CryptoCreateOutput` ?

Reply:
Good catch, I think I wrote this before BlockStream.
Will fix.